src/Features/OnboardingTasks/TaskLists.php

```diff
@@ -162,16 +162,16 @@ public static function get_task( $id, $task_list_id = null ) {
 			return null;
```

Comment:
The issue I mentioned in https://github.com/woocommerce/woocommerce-admin/pull/7727 is a couple lines up, with `self::get_task_list_by_id`, which doesn't exist in the class.

Reply:
Thanks!  Updated.